browser/tor/tor_profile_service_factory.cc

```diff
@@ -129,7 +129,8 @@ KeyedService* TorProfileServiceFactory::BuildServiceInstanceFor(
         g_brave_browser_process->tor_pluggable_transport_updater();
   }
   return new tor::TorProfileServiceImpl(
-      context, g_browser_process->local_state(), tor_client_updater,
+      Profile::FromBrowserContext(context)->GetOriginalProfile(), context,
```

Comment:
Noting that this is a Tor-related request **not** going over Tor, but it's intended in this case because Tor may not be accessible without a bridge. Also, it's the same request for all users.